tools/system_libs.py

```diff
@@ -49,14 +49,16 @@ def glob_in_path(path, glob_pattern, excludes=()):
   return sorted(f for f in files if os.path.basename(f) not in excludes)
 
 
-def get_base_cflags(force_object_files=False):
+def get_base_cflags(force_object_files=False, asm=False):
```

Comment:
What does "asm" mean in this context? I can't make sense of the name "asm" to the usage below which adds a dynamic linking flag.

Reply:
It means "assembly file"  .. i.e. `-D` and `-I` flags should not be added

Reply:
I renamed it to "preprocess" to be more clear.